cmd/root.go

```diff
@@ -170,13 +174,15 @@ There will be a ClusterPolicyReport with results for cluster-wide resources.`,
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute(rootCmd *cobra.Command) {
 	if err := rootCmd.Execute(); err != nil {
-		log.Fatal().Err(err).Msg("Error on cmd.Execute()")
+		slog.Log(context.Background(), logconfig.LevelFatal, "Error on cmd.Execute()", "error", err)
```

Comment:
This is a logger is a setup CLI logger. Therefore we can use `log.Fatal` for simplicity as we called `slog.SetDefault` in the `RunE` function above.

Reply:
Using the `log` std library package is being yelled at by the linter because we're also using `slog` in the same file. I have added a `//nolint` to make linter less fierce, but it needs to be reviewed.